interface/src/app/analyze/analysis/analysisDetail.tpl.html

```diff
@@ -73,10 +73,62 @@
         <td>{{sb.sampleData[id].name}}</td>
         <td ng-repeat="at in analysis.exampleCols">{{
           sb.sampleData[id].annotations[at.typename]
-          }}</td>
+        }}</td>
       </tr>
     </tbody>
   </table>
+  <div ng-if="sb.samples.length">
+    <h4>Samples Missing Activity</h4>
+    <p>The samples below are missing activity data, either because raw data
+      are unavailable or because they have been omitted from the ADAGE
+      compendium for another reason. Because data are not available, these
+      samples cannot be included in further analysis.
+    </p>
+    <button type="button" class="btn btn-default btn-sm"
+            ng-click="sb.clearSamplesMissingActivity()"
+            uib-tooltip="Remove all samples missing activity data"
+            tooltip-placement="right">
+      <i class="fa fa-times" aria-hidden="true"></i> Remove
+    </button>
+    <table class="table table-hover">
+      <thead>
+        <tr>
+          <th>&nbsp;</th>
+          <th>
+            <span class="row-control">
+              <i class="fa fa-bars" aria-hidden="true"></i>
+            </span>
+            ML Data Source
+          </th>
+          <th>id</th>
+          <th>Sample Name</th>
+          <th ng-repeat="at in analysis.exampleCols">{{
+            at.typename
+          }}</th>
+        </tr>
+      </thead>
+      <tbody ng-model="sb.samples">
+        <tr ng-repeat="id in sb.samples"
+            ng-mouseover="showControls = true"
+            ng-mouseleave="showControls = false">
+          <td class="row-control">
+            <button type="button" class="btn row-control"
+                    ng-click="sb.removeSample(id)" aria-label="Delete">
+              <i class="fa fa-times" aria-hidden="true"></i>
```

Comment:
Hmm, this `x` button seems too muted to me. I totally missed it the first time I looked at the table (even when the row is highlighted). The user needs to hover directly over it for it to be a bit more clear. Is this the same issue as with the table above, where it would be very CSS-involved to fix it?

Reply:
Agree. Can we use a more obvious button here?

Reply:
This is exactly the same issue as above. In fact, I used a modified version of the same table markup here. As part of (near) future refactoring work this will become a `sampleTable` component in which I am planning to eliminate the need for this column of controls, so I'd like to defer tweaking, if I may, until we get to #285.

Although I expect this related refactoring to come soon it is not on the list for our October release. I agree it is a usability issue, so if you feel it should happen sooner, let's raise the priority of #285 or create a separate issue to revisit the CSS headache that currently haunts those controls.

Reply:
I think this is important to user experience. Can we boot its priority?

Reply:
Which approach do you want to take, @dongbohu? Work on a temporary solution via CSS or start working on #285 sooner? I am fine with going either way, but I'm not certain I could complete #285 before the end of this month without delaying another issue on my list.

Reply:
What about putting #285 in next month release?

Reply:
Totally happy with that way of doing it. I'll have to take a closer look toward the end of the month. I may want to break that issue into smaller pieces before tackling it, but right now my gut says that if it were top priority it would fit into the next release.